scripts/sanitycheck

```diff
@@ -2644,10 +2644,13 @@ class TestSuite:
             total_tests = 0
             with open(file, "r") as fp:
                 cr = csv.DictReader(fp)
+                # Skip header
+                next(cr)
```

Comment:
Is this really needed, I thought the DictReader will pull off the header?

Reply:
it is not, with the above change